### DISCUSSION TITLE
No such file or directory

### DISCUSSION THREAD

**Discussion by kwarteng10** [idaholab/moose#17611] [Q&A Getting Started] [answered]
Hi, I got MOOSE installed and i tried creating an application with the command: cd ~/projects
However, i got an error message saying: bash: cd: /root/projects: No such file or directory 
I checked the MOOSE folder and there is a file named projects
Any help as to how to get this solved?
Thank you

**Comment by GiudGiud** [comment, marked as answer]
Hello

`cd ~/projects` means change directory to projects in the home (~) directory.
Except you created the projects directory in the moose directory instead of the home directory.

You could restart the tutorial from the beginning to correct that.
We recommend the directory structure to be something like:
```
(moose) guillaume@ubuntu1:~/projects$ pwd
/home/guillaume/projects
(moose) guillaume@ubuntu1:~/projects$ ll
total 136
drwxrwxr-x 18 guillaume guillaume  4096 Apr 13 18:54 ./
drwxr-xr-x 29 guillaume guillaume  4096 Apr 14 22:47 ../
drwxrwxr-x 20 guillaume guillaume  4096 Apr 13 19:03 moose/
drwxrwxr-x 20 guillaume guillaume  4096 Apr 13 19:03 your_application/
```
with moose and your application in the projects folder under the home directory.

This is not a moose issue, you will find more help on linux help forums. 

Guillaume

**Comment by friedmud** [comment]
There seems to be a bigger issue than that... @kwarteng10 are you using the `root` user?  It's really not a good idea.  I would suggest adding a "regular" user and logging in with that.

**Reply by kwarteng10** [reply]
How do I add a regular user
Sorry I am new to Ubuntu, so I seem to be lost

**Comment by kwarteng10** [comment]
I am running ubuntu on windows sir

**Comment by kwarteng10** [comment]
You mean I am running from a wrong directory?

**Reply by kwarteng10** [reply]
I got it now
It is working just fine
